packages/web-twig/tests/ComponentTest.php

```diff
@@ -25,6 +25,7 @@ public function setUp(): void
      */
     public function testShouldRender(string $testTemplate, string $prefix = null): void
     {
+        $this->markTestSkipped('Here is a namespace collision with `ui-component`.');
```

Comment:
@tomasjanicek 
```
make phpunit
> vendor/bin/phpunit
PHPUnit 9.6.6 by Sebastian Bergmann and contributors.

Warning:       XDEBUG_MODE=coverage or xdebug.mode=coverage has to be set

..........................................................E....  63 / 140 ( 45%)
E.............................................................. 126 / 140 ( 90%)
..............                                                  140 / 140 (100%)

Time: 00:03.337, Memory: 34.00 MB

There were 2 errors:

1) Lmc\SpiritWebTwigBundle\Resources\components\Button\ButtonSnapshotTest::test with data set "buttonLoadingDefault.twig" ('buttonLoadingDefault.twig')
Twig\Error\LoaderError: There are no registered paths for namespace "ui-component" in "@spirit/Button/Button.twig" at line 36.

/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Loader/FilesystemLoader.php:204
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Loader/FilesystemLoader.php:131
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Environment.php:264
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:319
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:379
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/TemplateWrapper.php:40
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Environment.php:280
/srv/spirit-web-twig-bundle/tests/AbstractComponentSnapshotTest.php:39
phpvfscomposer:///srv/spirit-web-twig-bundle/vendor/phpunit/phpunit/phpunit:106

2) Lmc\SpiritWebTwigBundle\Resources\components\ButtonLink\ButtonLinkSnapshotTest::test with data set "buttonLinkLoadingDefault.twig" ('buttonLinkLoadingDefault.twig')
Twig\Error\LoaderError: There are no registered paths for namespace "ui-component" in "@spirit/ButtonLink/ButtonLink.twig" at line 36.

/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Loader/FilesystemLoader.php:204
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Loader/FilesystemLoader.php:131
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Environment.php:264
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:319
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:394
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:367
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Template.php:379
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/TemplateWrapper.php:40
/srv/spirit-web-twig-bundle/vendor/twig/twig/src/Environment.php:280
/srv/spirit-web-twig-bundle/tests/AbstractComponentSnapshotTest.php:39
phpvfscomposer:///srv/spirit-web-twig-bundle/vendor/phpunit/phpunit/phpunit:106

ERRORS!
Tests: 140, Assertions: 141, Errors: 2.
Script vendor/bin/phpunit handling the phpunit event returned with error code 2
make: *** [phpunit] Error 2
```

Reply:
I do not know why but this test somehow overrides the namespace and Button and ButtonLink are not found.

Reply:
Here is mine:

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/4bd4b038-49b0-4777-96a8-bf5105b3dbac)

Reply:
Yeah, I have added the skip so that is why there are those `SSSSS`.

Reply:
This has been solved by modifying `phpunit.dist.xml` and running the directories with tests in the test suite in serial.